java/client/src/main/java/glide/api/models/configuration/BaseClientConfiguration.java

```diff
@@ -74,4 +74,7 @@ public abstract class BaseClientConfiguration {
      * used.
      */
     private final Integer inflightRequestsLimit;
+
+    /** TBD */
```

Comment:
TBD

Reply:
I could make a description based on the official definition of what an az is:
> A distinct location within a Region that is insulated from failures in other [Availability](https://wa.aws.amazon.com/wellarchitected/2020-07-02T19-33-23/wat.concept.availability.en.html) Zones, and provides inexpensive, low-[latency](https://wa.aws.amazon.com/wellarchitected/2020-07-02T19-33-23/wat.concept.latency.en.html) network connectivity to other [Availability](https://wa.aws.amazon.com/wellarchitected/2020-07-02T19-33-23/wat.concept.availability.en.html) Zones in the same Region.